lib/server.js

```diff
@@ -15,7 +15,9 @@
 
 'use strict';
 
-var _ = require('lodash');
+var isString = require('lodash/isString');
```

Comment:
what is the advantage in selecting modules inside loads? instead of just keeping `var _ = require('lodash');`?